doc/users/saving_workflows.rst

```diff
@@ -73,27 +73,27 @@ This will create a file "outputtestsave.py" with the following content:
 	    return out
 
 	testfunc.inputs.function_str = getsource(testfunc_1)
-	testfunc.inputs.ignore_exception = False
+	testfunc.ignore_exception = False
```

Comment:
.interface?

Reply:
Yup, and I think that the class attribute does not exist either.